syntax_checkers/ebuild/pkgcheck.vim

```diff
@@ -0,0 +1,57 @@
+" Syntax checking plugin for syntastic
+" Language:	Gentoo Ebuilds/Eclasses
+" Author:	Anna Vyalkova <cyber+gentoo@sysrq.in>
+" Copyright:	Copyright (c) 2022 Anna Vyalkova
+" Licence:	You may redistribute this under the same terms as Vim itself
+"
+" Syntax checker for ebuilds and eclasses powered by pkgcheck.
+" Requires vim 7.0.175 or later.
+"
+
+" keep in sync with Syntastic
+if &compatible || v:version < 700 || (v:version == 700 && !has('patch175'))
```

Comment:
Doesn't syntastic expose some way of doing these checks without having to inline them?

Reply:
Yes, it does these checks itself. Removed.